APIs/src/EpiServer.ContentGraph/Api/Querying/GraphQueryBuilder.cs

```diff
@@ -42,6 +42,14 @@ public static GraphQueryBuilder CreateFromConfig(Action<OptiGraphOptions> transf
                 var httpClientFactory = ServiceLocator.Current.GetService(typeof(IHttpClientFactory)) as IHttpClientFactory;
                 if (transformAction != null)
                 {
+                    if (options.TransformActionBehaviour == TransformActionBehaviour.Clone)
+                    {
```

Comment:
The reason I added the TransformActionBehaviour is to avoid changing the behaviour which could potentially be a breaking change. By keeping the Default behaviour a customer can upgrade an existing solution without configuring this option. 
But I recognize that this might be unnecessary depending on which use cases that exists. Please keep in mind and consider during review.